### PR TITLE
Newark styling

### PR DESCRIPTION
**Pull request by maysakanoni**

### CODE REVIEW

**Comment by maysakanoni** [comment]
[Apply header and footer to Newark service](https://trello.com/c/AJwPXgeu/102-apply-header-and-footer-to-newark-service)

**Comment by maysakanoni** [comment]
[Apply Newark generic styling](https://trello.com/c/ckQaYJH4/86-apply-newark-generic-styling)

File: styles/header.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.nsdc-header{
+  background-color: #fff;
+}
+
+.header-link:hover{
+  border-bottom: 3px solid black;
+}
+
+.border-banner {
+  background-image: url(https://assets.hexcdn.uk/nsdc/assets/img/border-banner.svg);
```
Comment (markWarr) [inline styles/header.scss:10]: Can we guarantee that this image hosted by a 3rd party will always be available?

File: tests/cypress/integration/landingPage.spec.js [hunk 2]
```diff
@@ -23,7 +19,11 @@ describe('App', () => {
       'have.contain',
       'For other emergency repairs, please see customer services contact details page.'
     );
-    cy.get('[data-testid=landing-page-emergency-warning] a').should('have.attr', 'href', 'https://www.lincoln.gov.uk/contact-1/get-touch');;
+    cy.get('[data-testid=landing-page-emergency-warning] a').should(
+      'have.attr',
+      'href',
+      'https://www.lincoln.gov.uk/contact-1/get-touch'
```
Comment (Renny4Real) [inline tests/cypress/integration/landingPage.spec.js:25]: Should this point to a lincoln address?

**Review by Renny4Real — COMMENTED** [review]
Just a question 🙏🏾

File: tests/cypress/integration/landingPage.spec.js (hunk 2 above)
Comment (maysakanoni) [reply tests/cypress/integration/landingPage.spec.js]: yes it should!

File: pages/_app.js [hunk 3]
```diff
@@ -1,77 +1,48 @@
...
+    window.GOVUKFrontend.initAll();
+  };
+  useEffect(enableJavascript, []);
 
-  return <>
-    <header className="govuk-header " role="banner" data-module="govuk-header">
-      <div className="govuk-header__container govuk-width-container">
-        <div className="govuk-header__logo">
-          {/* eslint-disable-next-line @next/next/no-html-link-for-pages */}
-          <a href="https://www.lincoln.gov.uk/" className="govuk-header__logotype" target="_blank" rel="noopener noreferrer"/>
```
Comment (MK-NG) [inline pages/_app.js:20]: Should we be opening this in a new tab with `target="blank"` ? It's inconsistent from other services on NSDC. 

We should also add an aria-label like
`aria-label="Neward and Sheerwood Council Home page"` or something similar

File: tests/cypress/integration/landingPage.spec.js [hunk 4]
```diff
@@ -38,17 +38,30 @@ describe('App', () => {
...
 
   it('displays a start button', () => {
-    cy.get('a').contains('Start now').should('have.attr', 'href', '/report-repair/priority-list');
+    cy.get('a')
+      .contains('Start now')
+      .should('have.attr', 'href', '/report-repair/priority-list');
   });
 
   it('has an accessibility link', () => {
-    cy.get('a').contains('Accessibility').should('have.attr', 'href', 'https://www.lincoln.gov.uk/accessibility');
```
Comment (MK-NG) [inline tests/cypress/integration/landingPage.spec.js:52]: Can this link text be 'Accessibilty statement' for a bit more context. 

Can these links in the footer aslo have a `govuk-footer__link` class attribute so they're styled as such.

File: compoments/header.js [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+  return (
+    <>
+      <header
+        className="nsdc-header govuk-header"
+        role="banner"
+        data-module="govuk-header"
+      >
+        <div className="govuk-header__container govuk-width-container">
+          <div className="govuk-header__logo">
+            <a
```
Comment (MK-NG) [inline compoments/header.js:13]: Can we add an aria-label to this link.

`aria-label="Newark and Sheerwood Council Home Page"` for example.

File: pages/_app.js [hunk 6]
```diff
@@ -1,77 +1,48 @@
...
+      <Header></Header>
+      <div className="govuk-width-container">
+        <main className="govuk-main-wrapper govuk-!-padding-0">
+          <Component {...pageProps} err={err} />
+        </main>
+      </div>
+      <footer className="govuk-footer " role="contentinfo">
+        <div className="govuk-width-container ">
+          <Link className="footer-custom-link" href="">
+            Accessibility
```
Comment (MK-NG) [inline pages/_app.js:26]: Can content be changed to 'Accessibility Statement' to add a bit more context.

Can these footer links also include `govuk-footer__link` class attribute so they're styled as such - think that can replace the `footer-custom-link`class

File: styles/header.scss (hunk 1 above)
Comment (maysakanoni) [reply styles/header.scss]: I've got a local copy of it now

**Comment by MK-NG** [comment]
The footer links don't seem to be rendering the classes

Screenshot 2022-07-28 at 10 08 45

There's also unnecccessary closing and opening of `govuk-width-container` 
Can we also put the links in a list? 

Following code should be good: 

`<footer class="govuk-footer " role="contentinfo">
<div class="govuk-width-container ">
  <h2 class="govuk-visually-hidden">Support links</h2>
  <ul class="govuk-footer__inline-list">
  <li class="govuk-footer__inline-list-item">
    <a class="govuk-footer__link" href="">Accessibility Statement</a></li>
  <li class="govuk-footer__inline-list-item">
    <a class="govuk-footer__link" href="https://www.newark-sherwooddc.gov.uk/privacynotice/">Privacy</a> </li> 
  </ul> 
</div>
</footer>`

**Review by markWarr — APPROVED** [review]

**Review by markWarr — APPROVED** [review]

**Pull request merged**